enterprise/backend/src/metabase_enterprise/serialization/dump.clj

```diff
@@ -24,11 +25,41 @@
 
 (set! *warn-on-reflection* true)
 
+(def ^:private serialization-order
+  (delay (-> (edn/read-string (slurp (io/resource "serialization-order.edn")))
+             (update-keys name)
+             (update-vals (fn [order]
+                            (into {} (map vector order (range))))))))
+
+(def ^:private serialization-sorted-map
```

Comment:
Unless the function being memoized is really trivial, we tend to write these as
```clojure
(defn- sorted-map-for-model* [model]
  #_implementation-goes-here)

(def ^:private sorted-map-for-model (memoize sorted-map-for-model*))
```

Reply:
Done